src/plugins/vis_type_vega/public/data_model/vega_parser.test.js

```diff
@@ -58,8 +58,28 @@ describe(`VegaParser._setDefaultColors`, () => {
     `vegalite`,
     check({}, true, {
       config: {
+        axis: {
+          domainColor: '#eef0f3',
+          gridColor: '#eef0f3',
+          tickColor: '#eef0f3',
+        },
+        background: 'transparent',
         range: { category: { scheme: 'elastic' } },
         mark: { color: '#54B399' },
+        style: {
+          'group-title': {
+            fill: '#69707d',
+          },
+          'guide-label': {
+            fill: '#69707d',
+          },
+          'guide-title': {
+            fill: '#69707d',
+          },
+        },
+        title: {
+          color: '#343741',
```

Comment:
Instead of checking the color code directly, that may change in the future, what if we just check if we get the exact color assignment that we expect? something like that: 
```
domainColor: euiThemeVars.euiColorChartLines,
gridColor: euiThemeVars.euiColorChartLines,
tickColor: euiThemeVars.euiColorChartLines,
```

Reply:
done